src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/RuntimeEventSource.cs

```diff
@@ -50,6 +50,12 @@ public static void Initialize()
         // as you can't make a constructor partial.
         private RuntimeEventSource(int _) { }
 
+        [Event(2, Level = EventLevel.Informational)]
+        internal void LogAppContextSwitch(string switchName, int value)
+        {
+            base.WriteEvent(2, switchName, value);
+        }
```

Comment:
```suggestion
        [Event(1, Level = EventLevel.Informational)]
        internal void LogAppContextSwitch(string switchName, int value)
        {
            base.WriteEvent(1, switchName, value);
        }
```

It doesn't matter functionally, but I think this is the first event on this source, so it should get ID 1.